l10n_th_tin_service/views/res_partner_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_partner_form_move_vat" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
```

Comment:
name_company is in l10n_th_partner module, I think this view should inherit on that.
```suggestion
        <field name="inherit_id" ref="l10n_th_partner.view_partner_form" />
```